.happy/terraform/modules/sfn/main.tf

```diff
@@ -195,6 +195,69 @@ resource "aws_sfn_state_machine" "state_machine" {
 EOF
 }
 
+
+resource "aws_sfn_state_machine" "state_machine_seurat" {
+  name     = "dp-${var.deployment_stage}-${var.custom_stack_name}-seurat-sfn"
+  role_arn = var.role_arn
+
+  definition = <<EOF
```

Comment:
Should add a comment indicating this step function temporarily lives in 2 repos: infra and portal